Saudox/resources/views/profissional/anamnese/psicopedagogia/criar.blade.php

```diff
@@ -1,5 +1,1161 @@
+                            <label>Meses do ano</label><input value="Meses do ano" type="checkbox" name="conhece_tais_coisas[]" id="mesmo_sexo" {{ in_array_old($anamnese, "Meses do ano", "conhece_tais_coisas") }}><br>
+                            <label>Saber recortar</label><input value="Saber recortar" type="checkbox" name="conhece_tais_coisas[]" id="mesmo_sexo" {{ in_array_old($anamnese, "Saber recortar", "conhece_tais_coisas") }}><br>
+                            <label>Saber dias da semana</label><input value="Saber dias da semana" type="checkbox" name="conhece_tais_coisas[]" id="mesmo_sexo" {{ in_array_old($anamnese, "Saber dias da semana", "conhece_tais_coisas") }}><br>
+                            <br><br><br>
+
+                            <h3>Tem tique? Quais?</h3>
+                            <input type="text" name="apresenta_tiques_se_sim_quais" value="{{ old2($anamnese, "apresenta_tiques_se_sim_quais") }}" />
+
+                            <h3>Como pega no lapis?</h3>
+                            <input type="text" name="como_pegua_o_lapis" value="{{ old2($anamnese, "como_pegua_o_lapis") }}" />
+
+                            <h3>Força da escrita</h3>
+                            <input type="text" name="forca_da_escrita" value="{{ old2($anamnese, "forca_da_escrita") }}" />
+
+                            <h3>Como vocês acham que começou o problema? A que fatores atribuem?</h3>
+                            <textarea class="textareas_form" name="como_acha_que_surgiu_o_problema_a_que_fatores_atribuem" rows="4" cols="50" style="">{{ old2($anamnese, "como_acha_que_surgiu_o_problema_a_que_fatores_atribuem") }}</textarea>
+                            <br><br><br>
+
+                            <h3>Outras questões</h3>
+                            <textarea class="textareas_form" name="outras_questoes" rows="4" cols="50" style="">{{ old2($anamnese, "outras_questoes") }}</textarea><br><br><br>
+
+                            <h3>Quais escolas frequentou e quando as frequentou?</h3>
+                            <textarea class="textareas_form" name="escolas_que_frequentou" rows="4" cols="50" style="">{{ old2($anamnese, "escolas_que_frequentou") }}</textarea><br><br><br>
+
+                            <h3>Repetiu de ano? Quando o porque?</h3>
+                            <input type="text" name="repetiu_ano_se_sim_qual_e_porque" value="{{ old2($anamnese, "repetiu_ano_se_sim_qual_e_porque") }}" />
+
+
+
+
+
+
+
+                        </div>
+                        <div class="pag_form">
+                            <br>
+                            <br>
+                            <hr class="hr_form">
+                            <h1>Comportamento</h1>
+                            <br>
+
+                            <h3>Humor habitual</h3>
+                            <input type="text" name="humor_habitual" value="{{ old2($anamnese, "humor_habitual") }}" />
+
+                            <h3>Brinca sozinho ou em grupo?</h3>
```

Comment:
```suggestion
                            <h3>Prefere brincar sozinho ou em grupo?</h3>
```
![image](https://user-images.githubusercontent.com/25617786/95791313-7e278600-0cb7-11eb-9254-0c765c7d4236.png)